docker-compose.yml

```diff
@@ -0,0 +1,90 @@
+version: "3.4"
```

Comment:
`version` se vise ne koristi
> docker-compose.yml: the attribute `version` is obsolete, it will be ignored, please remove it to avoid potential confusion

```suggestion
```